### ISSUE TITLE
Fetches only 100 projects per project-group

### ISSUE THREAD

**Issue by necixy** [itsTeknas/GitlabBackupUtil#8]
If one gitlab project-group contains more than 100 projects, it fetches only 100 projects. Although on API call it was mentioned per_page=999 but I guess this doesn't work.

**Comment by itsTeknas** [comment]
Any known workarounds ?

**Comment by necixy** [comment]
Yes, I search for that and found that for sure provide next_page which we can use to loop until there's no further page. I'll also check if I can get some time and code that and push that.

**Comment by itsTeknas** [comment]
That would be great !
